chevah_build

```diff
@@ -386,8 +383,15 @@ case $OS in
         export BUILD_LIBEDIT="no"
         ;;
     rhel5|sles11)
-        # OS'es with OpenSSL 0.9.8 libs, older cryptography needed here.
-        PIP_LIBRARIES=$PIP_LIBRARIES_OPENSSL_098
+        # OS'es with OpenSSL 0.9.8 libs, older cryptography would be needed,
+        # but upstream provides a Linux wheel that works, at least on SLES 11.
```

Comment:
it talks about "at least SLES11" but I see that this is also applied to RHEL5.

I think that we should not mention the SLES 11 but rather explain why the upstream wheel works.

Reply:
Yes, I can add here a message about the bundled OpenSSL as well.

I wrote _at least on SLES 11_ because that's what I tested. I haven't tested this on RHEL 5, so I'm not sure it will work. Hopefully we'll never need to find out. :-]